### PR TITLE
Add botamusique

### PR DESCRIPTION
**Pull request by 0boro**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -10,4 +10,7 @@ services:
     environment:
       TZ: Europe/Berlin
     restart: unless-stopped
-
+  botamusique:
+    build: plugins/botamusique/.
+    ports:
+      - 8181:8181
```
Comment (cfstras) [inline docker-compose.yaml]: what's this port doing?

File: plugins/botamusique/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+
+FROM python:alpine
```
Comment (cfstras) [inline plugins/botamusique/Dockerfile]: pin base image to a version

File: plugins/botamusique/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+ARG VERSION=6.0
+
+RUN apk add git zlib-dev libjpeg-turbo-dev gcc musl-dev
+RUN apk add opus opus-dev libmagic ffmpeg
```
Comment (cfstras) [inline plugins/botamusique/Dockerfile]: merge all `apk add` into one RUN command
you can split the line:
```docker
RUN apk add this that \
    other things
```

File: plugins/botamusique/configuration.ini [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+autoplay_length = 5
+# target version, stable or testing (testing need to bot installed with git)
+target_version = stable
+# Users allowed to kill the bot, or ban URLs.
+admin = 0boro
+# Folder that stores your local songs.
+music_folder = music_folder/
+# Folder that stores the downloaded music.
+tmp_folder = /tmp/
+database_path = database.db
```
Comment (cfstras) [inline plugins/botamusique/configuration.ini]: add a volume (to dockerfile _and_ docker-compose file) to persist the database?

File: plugins/botamusique/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+RUN git clone --branch ${VERSION} --recurse-submodules https://github.com/azlux/botamusique.git
```
Comment (cfstras) [inline plugins/botamusique/Dockerfile]: why not a submodule? lets us pin the exact revision

**Review by cfstras — CHANGES_REQUESTED** [review]
I have some feedback

**Comment by 0boro** [comment]
sry for my `git commit --amend` behaviour, someone needs to exorcise Gerrit of me.

**Review by LennyPenny — APPROVED** [review]

**Pull request merged**